AuxTel/Standard-Operations/Daytime-Operations/latiss-daily-calibrations-BIAS-DARK-only-procedure.rst

```diff
@@ -0,0 +1,54 @@
+.. _`RubinTV`: https://summit-lsp.lsst.codes/rubintv/summit/auxtel 
+.. _cp_verify: https://github.com/lsst/cp_verify
+.. _butler: https://pipelines.lsst.io/v/daily/modules/lsst.daf.butler/index.html
+.. _collection: https://pipelines.lsst.io/v/daily/modules/lsst.daf.butler/organizing.html
+.. _BLOCK source code: https://github.com/lsst-ts/ts_config_ocs/blob/develop/Scheduler/observing_blocks_auxtel/block-295-latiss_daily_calibrations.json
+.. _AuxTel (LATISS) Temperatures and Pressures dashboard: https://summit-lsp.lsst.codes/chronograf/sources/1/dashboards/14
+
+.. |author| replace:: *Karla Peña Ramírez*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *OS team*
+
+.. _Daytime-Operations-LATISS-Daily-Calibrations-BIAS-DARK-only-Procedure:
+############################################
+LATISS Daily Calibrations BIAS and DARK only
+############################################
+
+.. _Daytime-Operations-LATISS-Daily-Calibrations-BIAS-DARK-only-Overview:
+Overview
+========
+This procedure will acquire BIAS and DARK calibrations images. After the entire script completion, the observer can decide whether to vent, proceed to on-sky or shutdown.
```

Comment:
Would this be "calibration images" (singular) rather than "calibrations images"?